src/content/documentation/publish/source-code-submission.md

```diff
@@ -63,15 +63,22 @@ To reproduce the build, the reviewer runs the instructions you provided and then
 - details, including required version and installation instructions, of any tools or utilities that need to be downloaded, for example, [yuicompressor](http://yui.github.io/yuicompressor/).
 - a list of all the commands to generate an identical copy of the extension from the source code, for example, npm install or a grunt target. Ideally, you should include every command in the build script file.
 
+Reviewers will try to replicate your build environment if specified. However, if you do not specify the build environment, here's the default build environment that reviewers use:
+
+- Latest Ubuntu LTS (Desktop edition) virtualized via VirtualBox running on an x86_64 host
+- 10GB of system memory (RAM), 6 cores of vCPU
+- Latest Node LTS and corresponding npm
+- 50GB of disk space
+
+Should any of the above differ from your build environment, it is essential that you let the reviewers know about it via the README file in your source submission otherwise the build might not match and your submission may get rejected.
```

Comment:
Just to double-check: Is the README the (one) location we want that to be provided in?

Reply:
I was contemplating whether to add "Notes for reviewers" here, but didn't see it mentioned for anything source code submission related from before so didn't add. Ideally the reviewer should be made aware of the platform requirements before they have already downloaded the source code inside their VM, which makes "Notes for reviewers" a slightly better place for it. Thoughts on if we change our recommendation from README to "Notes for reviewers" or something better? 

FWIW, we do mention 3PL sources be added to "Notes for reviewers"

Reply:
I don't have a preference, we should pick what works best for reviewers (from the available options of course, we don't want to add an additional burden to developers by coming up with something entirely new). For me, it's more important that there is **exactly one (!)** (also not zero) location that both the developer and reviewers are aligned on, rather than forcing a particular location (until we have a dedicated way to provide that).

I personally like when all information is within the source zip, because it's immutable and contained with the code itself, rather than in a different place, but that is not necessarily the ideal state, as developers might need to update their instructions sometimes. Having to resubmit the entire source package in that case might be beneficial overall to ensure code and instructions are somewhat in sync but it is also more effort than to just update a text field on AMO.
However, if instructions are in multiple places, they tend to be out of sync and that's when ambiguity, inaccuracy and conflicting information lead to reproducing failures, rejections and back and forth with the developer.

Reply:
@dotproto for context, there is an [idea](https://mozilla-hub.atlassian.net/browse/AOP-154) to standardize this, which to me sounds valuable to do.

Reply:
I agree that while README isn't ideal, it does come with its own merits and I'd favor not changing it right now (it does come with the additional work for reviewers to switch environment after opening up the source package, but this PR doesn't affect/change that effort). Let's stick to README and we can work towards [standardizing it](https://mozilla-hub.atlassian.net/browse/AOP-154).

Reply:
Thanks for the additional context, @wagnerand.